src/Tools/ExternalAccess/OmniSharp/CodeActions/OmniSharpCodeAction.cs

```diff
@@ -10,6 +10,6 @@ namespace Microsoft.CodeAnalysis.ExternalAccess.OmniSharp.CodeActions
     internal static class OmniSharpCodeAction
     {
         public static ImmutableArray<CodeAction> GetNestedCodeActions(this CodeAction codeAction)
-            => codeAction.NestedCodeActions;
```

Comment:
Let's remove this API entirely. Now that we have a public API, O# can just use that. FYI @JoeRobich.

Reply:
Typically this would be performed in a follow-up to reduce the likelihood of intermediate breaking changes.

Reply:
wfm to do this in followup.  i'll make this obsolete.